src/mono/netcore/System.Private.CoreLib/src/LinkerDescriptor/System.Private.CoreLib.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <linker>
 	<assembly fullname="System.Private.CoreLib">
-	
+		<type fullname="Interop/Globalization">
+			<!-- Internal API used by tests only. -->
+			<method name="GetICUVersion" />
+		</type>
```

Comment:
would be nice to move this to the bottom of the file, there is already another test-only dependency there.

Reply:
I thought it was sorted alphabetically by class. Will update and move it down.

Reply:
Fixed.